src/components/Header/Header.tsx

```diff
@@ -1,18 +1,34 @@
 import cn from 'classnames';
 import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
 
 import logo from '@/assets/images/icons/nice-gadgets-logo.svg';
+import { changeBurgerState } from '@/features/burgermenu/burgerSlice';
 import { HeaderNavigationLinks } from '@/constants';
+import { Navigation } from '../Navigation';
```

Comment:
Is that how linter transferred the import to you? That's a very strange place for him to be.